### DISCUSSION TITLE
Welcome to gopt Discussions!

### DISCUSSION THREAD

**Discussion by tczorro** [theochem/gopt#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tczorro** [comment]
Week 2 Code Review target file: **[internal.py](https://github.com/theochem/gopt/blob/refactor/gopt/internal.py)**

**Reply by PaulWAyers** [reply]
GOpt code review for 4pm on Thursday.

**Comment by PaulWAyers** [comment]
I'm going to close the Discussions before the next meeting. We have a (secret) team for discussing Gopt, and I'd like to keep that team for doing the discussions. I'm mostly worried about inadvertantly advertising a Zoom link or something similar in a public repo.
